packages/fast_bitvector/fast_bitvector.0.1.2.2/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+synopsis: "A bitvector library"
+description: "Bitvector represented as bytes internally"
+maintainer: ["Stefan Muenzel <source@s.muenzel.net>"]
+authors: ["Stefan Muenzel <source@s.muenzel.net>"]
+license: "MPL-2.0"
+tags: ["bitvector" "bitset"]
+homepage: "https://github.com/engineeredabstraction/fast_bitvector"
+bug-reports: "https://github.com/engineeredabstraction/fast_bitvector/issues"
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/engineeredabstraction/fast_bitvector.git"
+x-maintenance-intent: ["(latest)"]
+depends: [
+  "dune" {>= "3.18"}
+  ( ("ocaml" {>= "4.14.0" & <"5.1.0"})
+  | ("ocaml" {>= "5.1.0" & < "5.4.0"} & "ocaml_intrinsics_kernel" { arch != "arm32" & arch != "x86_32" })
+  | ("ocaml" {>= "5.4.0"}
+    & "ocaml_intrinsics_kernel" { arch != "arm32" & arch != "x86_32" }
+    & "ppxlib_jane" {>= "v0.17.4"}
```

Comment:
This is not tested for now, but will appear as revdeps once the package is released